src/main/java/nl/novi/TechItEasy/services/Television.java

```diff
@@ -0,0 +1,31 @@
+package nl.novi.TechItEasy.services;
+
```

Comment:
Television is een Model. Hoort thuis in Models folder en niet in Services folder